library/jenkins

```diff
@@ -2,8 +2,8 @@
 # maintainer: Michael Neale <mneale@cloudbees.com> (@michaelneale)
 # maintainer: Carlos Sanchez <csanchez@cloudbees.com> (@carlossg)
 
-latest: git://github.com/jenkinsci/docker@15dc59d7dbd47da5259a50a9ebfa8895d594444f
-2.60.2: git://github.com/jenkinsci/docker@15dc59d7dbd47da5259a50a9ebfa8895d594444f
+latest: git://github.com/jenkinsci/docker@dec928fd4eadbbbbcd6a35a3ad2baa38b537e846
+2.60.3: git://github.com/jenkinsci/docker@dec928fd4eadbbbbcd6a35a3ad2baa38b537e846
 
-alpine: git://github.com/jenkinsci/docker@c24687d11870e6a408678a394c1298008773bf03
-2.60.2-alpine: git://github.com/jenkinsci/docker@c24687d11870e6a408678a394c1298008773bf03
+alpine: git://github.com/jenkinsci/docker@dec928fd4eadbbbbcd6a35a3ad2baa38b537e846
+2.60.3-alpine: git://github.com/jenkinsci/docker@dec928fd4eadbbbbcd6a35a3ad2baa38b537e846
```

Comment:
This changes the Alpine version to use the same Dockerfile as the Debian version. I see that you have a `Dockerfile-alpine`, but the official images do not have a way to specify a different Dockerfile; it needs to be in another folder or a different branch like it was previously.